src/main/java/io/confluent/connect/hdfs/DataWriter.java

```diff
@@ -462,6 +462,7 @@ public void close() {
         topicPartitionWriters.remove(tp);
       }
     }
+    assignment.clear();
```

Comment:
Can it be null?

Reply:
No, we iterate over `assignment` right before this statement.